code/_macros.dm

```diff
@@ -50,10 +50,36 @@
 
 #define isdrone(A) istype(A, /mob/living/silicon/robot/drone)
 
+#define isWrench(A) istype(A, /obj/item/weapon/tool/wrench)
```

Comment:
We don't use any of these, toss them out. 

we check tool qualities instead of the object type

Reply:
but. but. they are handy, i thought you simply dont have them

Reply:
i get it, ill fix it

Reply:
actually you might be right, they might be handy to have. But still check qualities

if the bolt turning quality is in its tool qualities list, then its a wrench